src/processor.ts

```diff
@@ -1,20 +1,22 @@
-import {lookupArchive} from "@subsquid/archive-registry"
+import { lookupArchive } from "@subsquid/archive-registry"
 import * as ss58 from "@subsquid/ss58"
-import {BatchContext, BatchProcessorItem, SubstrateBatchProcessor} from "@subsquid/substrate-processor"
-import {Store, TypeormDatabase} from "@subsquid/typeorm-store"
-import {In} from "typeorm"
-import {Account, Transfer} from "./model"
-import {BalancesTransferEvent} from "./types/events"
+import { BatchContext, BatchProcessorItem, SubstrateBatchProcessor } from "@subsquid/substrate-processor"
+import { Store, TypeormDatabase } from "@subsquid/typeorm-store"
+import { In } from "typeorm"
+import { Account, Transfer } from "./model"
+import { BalancesTransferEvent } from "./types/events"
+import { config } from "./config"
+import { handleAssetSwap, handleLiquidityAdded, handleLiquidityRemoved, handleBalanceTransfer } from './mappings/protocol'
 
 
-const processor = new SubstrateBatchProcessor()
-    .setDataSource({
-        // Lookup archive by the network name in the Subsquid registry
-        //archive: lookupArchive("kusama", {release: "FireSquid"})
+const DataSelection = { data: { event: true } } as const
 
-        // Use archive created by archive/docker-compose.yml
-        archive: lookupArchive('kusama', {release: 'FireSquid'} )
-    })
+
+const processor = new SubstrateBatchProcessor()
+    .setDataSource(config.dataSource)
```

Comment:
why were we still using kusama as a data source? Switched to amplitude archive node